Sources/Datadog/Core/SessionTracer.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import _Datadog_Private
+
+// TODO: RUMM-300 This should probably be a part of DDTracer
```

Comment:
no, i was thinking of using `Tracer` as a namespace

for example,
```swift
let session = makeASession()
DDTracer.trace(session) // trace(:) is a static function
```

it could be under `Datadog` namespace as well.
it just shouldn't be a free function for the sake of discoverability

Reply:
I see 👍. I think `Datadog.` is a better choice because of what I discussed [here](https://github.com/DataDog/dd-sdk-ios/pull/99#discussion_r423833722).

Reply:
I'd say let's see how to plug tracing and/or RUM in another PR and keep the scope of this PR only to general isa swizzling, what do you think?

Reply:
I remove this todo as we will revisit this class with the introduction of `tracer`